src/msg/async/ProtocolV2.cc

```diff
@@ -1441,7 +1437,7 @@ CtPtr ProtocolV2::handle_message() {
       message->get_type() == CEPH_MSG_OSD_OPREPLY) {
     utime_t ltt_processed_stamp = ceph_clock_now();
     double usecs_elapsed =
-        (ltt_processed_stamp.to_nsec() - ltt_recv_stamp.to_nsec()) / 1000;
+        (ltt_processed_stamp.to_nsec() - recv_stamp.to_nsec()) / 1000;
```

Comment:
Since you are touching this line anyway, it would be nice to make ProtocolV1 and ProtocolV2 consistent.  The former uses floating point division, the latter uses integer division but still assigns to double.  Let's pick one and stick to it.

Reply:
@idryomov since the change was being tested. i merged it anyway. but i created #43443 as a follow-up change to address your comment.